artifacts-wg/charter/charter.md

```diff
@@ -0,0 +1,44 @@
+# Purpose
+In the distributed world of cloud-native computing different artifacts and packages are used to transport configuration and code for the many services and capabilities that comprise and support workloads and applications. As an example, today ArtifactHub advertises that it can crawl about 20 such artifact types [1]. These "cloud-native" bundles add to the previous proliferation of package and artifact types for software dependencies such as Maven and npm and for system packages like deb and rpm.
+
+This abundance of package types and formats adds complexity and risk for cloud application developers and architects trying to provision and run cloud-native workloads. Different tools and controllers must be carefully learned and installed to bundle, unbundle, verify and deploy each artifact type. This complexity slows product development by users, impedes innovation and collaboration amongst projects, and increases risks of insecure configuration.
+
+To reduce this complexity and facilitate collaboration and innovation, WG Artifacts will gather stakeholders from many CNCF and open source projects offering packaging, distribution and deployment mechanisms for bundles of configuration and code.
```

Comment:
```suggestion
To reduce this complexity and facilitate collaboration and innovation, WG Artifacts will gather stakeholders from many CNCF and open source projects to offer packaging, distribution and deployment mechanisms for bundles of configuration manifests and source code.
```

Reply:
I'm not sure if your suggestion here captures the intent of the original text.

Reply:
I think there's a significant difference between "offering" and "to offer". The idea here is that the group will bring together projects that _already_ offer bundle and packaging capabilities but currently follow their own siloed patterns and practices. We intend to bring these groups closer together in spirit _and_ implementations.

I'm fine with the latter changes but I think we should leave "offering" as is.